.github/workflows/build_helm_svc_defns.yml

```diff
@@ -41,7 +68,8 @@ jobs:
           echo "TEMPLATE_REPO_GITHUB_LOCATION=$TEMPLATE_REPO_GITHUB_LOCATION" >> $GITHUB_ENV
           echo "Cloning convoy-helm-template"
           rm -rf $TEMPLATE_REPO_GITHUB_LOCATION
-          git clone https://github.com/TessellDevelopment/convoy-helm-template.git $TEMPLATE_REPO_GITHUB_LOCATION
+          template_version=$(yq '.convoy-helm-template' Chart.yaml)
```

Comment:
The value for `template_version` will be `null` in case the `yq` command does not find the key in file. Please use `--exit-status` option to set the exit status and check the same.